.storybook/config.js

```diff
@@ -73,8 +73,8 @@ configure(() => {
   require('./../packages/bpk-component-drawer/stories');
   require('./../packages/bpk-component-fieldset/stories');
   require('./../packages/bpk-component-form-validation/stories');
-  require('./../packages/bpk-component-grid/stories');
   require('./../packages/bpk-component-grid-toggle/stories');
+  require('./../packages/bpk-component-grid/stories');
```

Comment:
This file was reordered slightly by the sort function implemented in `create-component.js`. We think this is probably fine.